neo/Network/RPC/RpcServer.cs

```diff
@@ -776,8 +776,8 @@ private JObject ValidateAddress(string address)
         private JObject GetStateHeight()
         {
             var json = new JObject();
-            json["blockHeight"] = Blockchain.Singleton.Height;
-            json["stateHeight"] = Blockchain.Singleton.StateHeight;
+            json["blockheight"] = Blockchain.Singleton.Height;
+            json["stateheight"] = Blockchain.Singleton.StateHeight;
```

Comment:
We shouldn't modify this! It's dangerous, as many exchanges and projects have already used this json format.